openpower/package/alternate-binutils/alternate-binutils.mk

```diff
@@ -0,0 +1,56 @@
+################################################################################
+#
+# alternate-binutils
```

Comment:
assuming all of this is identical to the current "primary" toolchain

Reply:
seem my other comment